Source/Generators/LMCArraySignalGenerator.cc

```diff
@@ -394,7 +395,7 @@ namespace locust
 
 
 
-    double ArraySignalGenerator::GetFIRSample(int nFilterBinsRequired, double dtFilter, unsigned channel, unsigned element)
+    double ArraySignalGenerator::GetFIRSample(int nFilterBins, int nFilterBinsRequired, double dtFilter, unsigned channel, unsigned element)

```

Comment:
I think nFilterSize doesn't need to passed as an argument anymore to GetFIRSample